packages/app-store/sendgridothercalendar/pages/setup/index.tsx

```diff
@@ -0,0 +1,157 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useRouter } from "next/router";
+import { useState, useEffect } from "react";
+import { Check, X } from "react-feather";
```

Comment:
Can we import these from "Calcom/ui/Icon` just for constancy sake

Reply:
Done, thanks!